.travis.yml

```diff
@@ -1,22 +1,34 @@
 # https://docs.haskellstack.org/en/stable/travis_ci/
-# run on containerized infrastructure hopefully
-dist: trusty
-sudo: false
```

Comment:
`dist: trusty` moves into the `jobs:` matrix below
`sudo: false` is deprecated/ignored